### PR TITLE
[DX-1684] Release notes for 5.0.14 (attempt 2)

### PR DESCRIPTION
**Pull request by andyo-tyk**
DX-1684
<br>

---

### Preview Link
https://deploy-preview-5428--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/version-5.0/
https://deploy-preview-5428--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.0/
<br>

Second attempt, without the accidental inclusion of other changes...

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 1]
```diff
@@ -10,6 +10,61 @@ weight: 2
...
+<li>
+<details>
+<summary>Fixed display issue for API statistics</summary>
+
+Fixed an issue where API statistics were not being shown when using PostgreSQL and adding two or more tags in the Activity page
+</details>
+</li>
+<li>
+<details>
+<summary>Fixed issue with display of HTTP 429 status codes on the Activity page</summary>
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]: I understand the fixes are only in the UI not the APIs, right @andyo-tyk

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 2]
```diff
@@ -10,6 +10,61 @@ weight: 2
...
+<li>
+<details>
+<summary>Fixed display of graphs and requests counter on Tyk Classic Portal</summary>
+
+Fixed wrong graphs and incorrect requests counter on Tyk Classic Portal when using PostgreSQL
+</details>
+</li>
+<li>
+<details>
+<summary>Fixed Error Breakdown display issues with dates</summary>
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]:
```suggestion
<summary>Tyk Dashboard UI: fixed issues with the Error Breakdown display, specifically related to date handling</summary>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 3]
```diff
@@ -10,6 +10,61 @@ weight: 2
...
+<li>
+<details>
+<summary>Fixed issue with display of HTTP 429 status codes on the Activity page</summary>
+
+Fixed an issue where HTTP 429 status codes were not being shown on the Activity page when using PostgreSQL
+</details>
+</li>
+<li>
+<details>
+<summary>Fixed display of graphs and requests counter on Tyk Classic Portal</summary>
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]:
```suggestion
<summary>Tyk Classic Portal UI: Fixed display of graphs and requests counter</summary>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md (hunk 1 above)
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]:
```suggestion
<summary>Tyk Dashboard UI:  Fixed issue with display of HTTP 429 status codes on the Activity page</summary>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 4]
```diff
@@ -10,6 +10,61 @@ weight: 2
...
+This release fixes some display issues in Tyk Dashboard and Tyk Classic Portal when using PostgreSQL.
+
+### Changelog {#Changelog-v5.0.14}
+
+#### Fixed
+
+<ul>
+<li>
+<details>
+<summary>Fixed display issue for API statistics</summary>
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]:
```suggestion
<summary>Tyk Dashboard UI: Fixed display issue for API statistics</summary>
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md]: The changes are in the Dashboard itself, in terms of the data provided to the UI. There is no change to the Dashboard API.

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 5]
```diff
@@ -10,6 +10,61 @@ weight: 2
 
 ---
 
+## 5.0.14 Release Notes
+
+### Release Date 18th September 2024
+
+### Release Highlights
+
+This release fixes some display issues in Tyk Dashboard and Tyk Classic Portal when using PostgreSQL.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md:19]:
```suggestion
This release fixes some display issues in Tyk Dashboard and Tyk Classic Portal when using PostgreSQL.

Please note, the fixes in this patch release are unrelated to the [critical fixes of Tyk Gateway v5.0.14]({{< ref "product-stack/tyk-gateway/release-notes/version-5.0">}})
```

**Review by JoanCamosTyk — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.0.md [hunk 6]
```diff
@@ -12,6 +12,65 @@ aliases:
 
 ---
 
+## 5.0.14 Release Notes
+
+### Release Date 18th September 2024
+
+### Release Highlights
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.0.md:31]:
```suggestion
### Breaking Changes
**Attention:** Please read this section carefully.

There are no breaking changes in this release.

### Upgrade Instructions

This release is not tightly coupled with [Tyk Dashboard v5.0.14]({{<ref "/product-stack/tyk-dashboard/release-notes/version-5.0">}}) so you do not have to upgrade both together.


Go to the [Upgrading Tyk](https://tyk.io/docs/product-stack/tyk-gateway/release-notes/version-5.0/#upgrading-tyk) section for detailed upgrade instructions.

### Release Highlights
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md [hunk 7]
```diff
@@ -10,6 +10,61 @@ weight: 2
 
 ---
 
+## 5.0.14 Release Notes
+
+### Release Date 18th September 2024
+
+### Release Highlights
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md:25]:
```suggestion
### Upgrade Instructions

This release is not tightly coupled with [Tyk Gateway v5.0.14]({{< ref "product-stack/tyk-gateway/release-notes/version-5.0">}}) so you do not have to upgrade both together.


Go to the [Upgrading Tyk](https://tyk.io/docs/product-stack/tyk-gateway/release-notes/version-5.0/#upgrading-tyk) section for detailed upgrade instructions.


### Release Highlights
```

**Review by sharadregoti — APPROVED** [review]

**Pull request merged**